lib/cli/src/generators/SERVER/index.ts

```diff
@@ -0,0 +1,14 @@
+import fse from 'fs-extra';
+import { baseGenerator, Generator } from '../baseGenerator';
+
+const generator: Generator = async (packageManager, npmOptions, options) => {
+  const prefix = fse.existsSync('./src') ? '../src' : '../stories';
+  const stories = [`${prefix}/**/*.stories.json`];
```

Comment:
People could have stories in any location, correct? Maybe we should just prescribe one like we do in other generators.

Reply:
Simplified this.